src/utils/TodoFilter.ts

```diff
@@ -0,0 +1,27 @@
+import { FilterLink } from '../types/FilterLinkTypes';
+import { Todo } from '../types/Todo';
+
+export function preparedTodos(
+  arr: Todo[],
+  option: string,
+) {
+  const todos = [...arr];
```

Comment:
The `filter` method returns a copy of the array, so you don't have to do it manually